### DISCUSSION TITLE
how do i get rid of this line when typing and this underline

### DISCUSSION THREAD

**Discussion by Joe257213** [davatorium/rofi#1347] [Q&A]
![image](https://user-images.githubusercontent.com/84450523/124589621-bb393700-de49-11eb-98c4-49d8f81cce27.png)
please help !

**Comment by DaveDavenport** [comment]
See rofi-themes manpage.

For the matching underline (on latest 1.6.1):
```css
element-text { highlight: none; }
```
The text cursor cannot be removed. It would be extremely confusing for the user not knowing where they are in the text.

**Reply by Joe257213** [reply]
change text cursor  colour ? or make it transparent

**Reply by DaveDavenport** [reply]
no, not possible.

**Reply by Joe257213** [reply]
or just change the text cursor symbol
